ui/client/settings/index.js

```diff
@@ -395,6 +395,52 @@ export const BundleService_RemoveSettingFromBundleURL = function(parameters = {}
   let keys = Object.keys(queryParameters)
   return domain + path + (keys.length > 0 ? '?' + (keys.map(key => key + '=' + encodeURIComponent(queryParameters[key])).join('&')) : '')
 }
+/**
+ * 
+ * request: PermissionService_ListPermissionsByResource
+ * url: PermissionService_ListPermissionsByResourceURL
+ * method: PermissionService_ListPermissionsByResource_TYPE
+ * raw_url: PermissionService_ListPermissionsByResource_RAW_URL
+ * @param body - 
+ */
+export const PermissionService_ListPermissionsByResource = function(parameters = {}) {
+  const domain = parameters.$domain ? parameters.$domain : getDomain()
+  const config = parameters.$config
+  let path = '/api/v0/settings/permissions-list-by-resource'
+  let body
+  let queryParameters = {}
+  let form = {}
+  if (parameters['body'] !== undefined) {

```

Comment:
You could remove this if condition, if you move `let body = parameters['body']` after line 418.

Reply:
code generation ftw.. but yes, this could be improved 🤷